README.md

```diff
@@ -92,40 +92,73 @@ You'll also need to create a private key for the service account (it should down
-    - Push to your fork and see events coming in
-5. Follow the steps of "Development & tests" to get the server running
-    - It will fail if you don't have all the env variables defined
-6. In order for your Slack app to work, you need to match the settings to the production Slack app. Run the following command and copy the output into the
-slack apps App Manifest.
+1.  Create a [new Slack App][slack_app] that matches the settings of the production app
+    - The prompt will ask you to associate to a workspace (use the new workspace you made in step 2)
+1.  In order for your Slack app to work, you need to match the settings to the production Slack app. Run the following command and copy the output into the slack app's App Manifest, making sure to use https:// URLs instead of http:// ones.
+
+    ```shell
+    sed 's|<NGROK_URL>|https://<NGROK_INSTANCE>.ngrok.io|g' ./.slack-manifest.example
+    ```
+
+    If you have `jq` installed and wish to avoid some manual copy-pasting, the following mini-script will also achieve the same effect once `ngrok http` is running in another terminal:
 
     ```shell
-    sed 's|<NGROK_URL>|https://<Your Ngrok URL ID>.ngrok.io|g' ./.slack-manifest.example
+    sed "s|<NGROK_URL>|$( curl -s localhost:4040/api/tunnels | jq -r '.tunnels[0].public_url')|g" ./.slack-manifest.example | pbcopy
     ```
 
-    - Make sure to use https:// URLs instead of http:// ones
-    - Some of the settings will need to be verified before they get save
-      - This means that you will need to update your `.env` file with the settings from your Slack app
-      - Reload your server for the new env vars to apply and resend the verification payloads
-      - You will have to do this with multiple settings, thus, you will have to repeat reloading your server as you add new variables
-    - On your new Slack workspace begin a conversation with the bot
-    - You should see your localhost app respond with 200 status code
-    - Congratulations!
+    - Some of the settings will need to be verified before they get saved
+    - This means that you will need to update your `.env` file with the settings from your Slack app
+    - Reload your server for the new env vars to apply and resend the verification payloads
+    - You will have to do this with multiple settings, thus, you will have to repeat reloading your server as you add new variables
+
+1.  Create a new GitHub repository at `<YOUR_GITHUB_USERNAME>/eng-pipes-dev`.
+1.  Create a [new GitHub App](https://github.com/settings/apps/new)
+    - Set the webhook to your ngrok tunnel with the GH route (e.g. `<NGROK_INSTANCE>/webhooks/github`)
+    - Place the secrets in your `.env` (see [Setup Secrets](#setup-secrets) below)
+    - Go to the `Permissions & events` sidebar menu entry of the GitHub app configuration, and grant maximum non-`Admin` access (`Read and write` where possible, `Read only` everywhere else) for every line in `Repository permissions`
```

Comment:
That's one way to do it. :)

This isn't how we run in prod. Do we care? Should we at least say something about that?

Reply:
A note would be nice. People using eng-pipes may not have access to the exact permissions and these may change in the future

Reply:
Done.